recipes/elyra/meta.yaml

```
@@ -0,0 +1,69 @@
+{% set name = "elyra" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/0b/0d/7355f28392a40bca18a373c2f6c55306a3f345ea257afa6e3a074a6e9785/elyra-{{ version }}-py3-none-any.whl
```

You are repackaging this project, can't you build it from sdist instead?

---

I would recommend you to upload the sdist to pypi as well

---

but for now you can use the release already available on github
```suggestion
  url: https://github.com/elyra-ai/elyra/archive/v{{ version }}.tar.gz
```

---

the license seems to be already in the release on github, so you can delete the license file